### DISCUSSION TITLE
Unclear where the usernames for the contacts are coming from

### DISCUSSION THREAD

**Discussion by nixpulvis** [zed-industries/zed#8067] [Feature Requests]
I guess I assume they are GitHub handles that have signed in with Zed? So they are actually Zed usernames, just taken as the same as the name's we signed into GitHub with?

I think this could be a bit more clear.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
